src/pages/ChokKahYang/baseKY.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./base.scss"
+import EduList from "./eduList"
+import Profile from "./profile.jpg"
+
+export default class BaseKY extends React.Component {
```

Comment:
move these content to your landingky and remove this component

Reply:
Ky chok absent today